### ISSUE TITLE
Need support for single line geocode and issues with custom geocoder

### ISSUE THREAD

**Issue by chris-fox** [Esri/incident-import-tools#11]
In my use case we are trying to load the building permits coming from another permit system. These records have a parcel number that we are using to geocode the locations. Oakland county has provided a geocoder that can locate features using the centroid of a parcel based the parcel id via the single line location. Everything geocodes fine when I use the Geocode Address gp tool.

However, when I use the Configure Import Incidents (Address Table) tool, it only allows me to specify the address fields. I need to be specify a single line input.

In addition, when I changed the source code to hard code my single line input, "'Single Line Input' Parcel_Number VISIBLE NONE", it successfully matched everything, but the logic for determine if there was match incorrectly determine that nothing matched to the acceptable level.

Attached is the csv I am attempting to geocode along with the configuration file which has the urls to the geocoder and the public service I am attempting to write to.

[TestData.zip](https://github.com/Esri/incident-import-tools/files/1711348/TestData.zip)

**Comment by cosbyr** [comment]
@chris-fox @ScottOppmann The tool already accepts Single Line Address Inputs. For this use case you input the single line address field into the "Street Address Field" and leave the rest of address fields blank. I tried to make that clear with the tooltip for that tool. As far as it not geocoding features correctly I will take a look at that problem and get back to you.

**Comment by cosbyr** [comment]
@chris-fox @ScottOppman. Looks like there was a small change with how the geocode addresses tool handled single line inputs between 2.0 and 2.1. I've adjusted the script to handle Single Line Inputs better now. Also ran your TestData.zip through the process and it imported successfully. This was a good test set... it introduced another problem I hadn't noticed before. Your test data had a "Status" field in it.... it just so happens that the Geocode GP tool also creates a "Status" field as part of its process.... this was causing some weird behavior in my code. I've addressed it now.
